python/foglamp/services/common/microservice.py

```diff
@@ -59,6 +59,7 @@ def __init__(self):
                     'description': 'Restrict microservice to localhost',
                     'type': 'boolean',
                     'default': 'false',
+                    'displayName': 'Restrict Microservice To Local'
```

Comment:
should  not this be  plural  for Microservice in `description` and `displayName`?